### PR TITLE
A page all about hot dogs for people who like hot dogs

### PR DESCRIPTION
**Pull request by snack-attack**
Add this branch if you like hot dogs. If you don't like hot dogs, you shouldn't be reviewing this code.

### CODE REVIEW

File: hotdogs.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
```
Comment (mporam) [inline hotdogs.html]: this is not great

File: hotdogs.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (mporam) [inline hotdogs.html]: this is good :)

File: hotdogs.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="">
+    </head>
+    <body>
```
Comment (mporam) [inline hotdogs.html]: your weird

File: hotdogs.html (hunk 1 above)
Comment (snack-attack) [reply hotdogs.html]: hot dogs ARE great!!!

File: hotdogs.html (hunk 3 above)
Comment (snack-attack) [reply hotdogs.html]: thanks, weird is good

**Pull request merged**